scripts/ci-push.sh

```diff
@@ -6,6 +6,17 @@ source ./scripts/ci-login.sh
 
 ./scripts/build-site.sh
 ./scripts/sync-and-test-bucket.sh
-./scripts/publish-sentry-release.sh
-./scripts/run-pulumi.sh update
-./scripts/make-s3-redirects.sh
+
+if [ -n "${SENTRY_AUTH_TOKEN:-}" ]; then
+  echo "Publishing Sentry release..."
+  ./scripts/publish-sentry-release.sh
+else
+  echo "Sentry auth token not found. Will skip publishing release to Sentry."
+fi
+
+if [ -n "${PULUMI_STACK_NAME:-}" ]; then
```

Comment:
We don't seem to be setting this anywhere. (I see it's defined as a secret, but not exported -- although really, shouldn't it be defined as a simple environment variable that's captured in code?)

Reply:
Yeah it's set as a GitHub Secret. But I can move it to the workflow file itself since it is not really a secret. But doing so would mean that if I wanted to fork this repo, I'd have to modify the workflow file, rather than set the GHA Secret.